tests/src/Kernel/CommerceStockKernelTestBase.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Drupal\Tests\commerce_stock\Kernel;
+
+use Drupal\Tests\commerce\Kernel\CommerceKernelTestBase;
+
+/**
+ * Class CommerceStockKernelTestBase.
+ *
+ * Provides a base class for commerce stock and submodules.
+ */
```

Comment:
Class docblock can be just one line: "Base class for Commerce Stock kernel tests."